### PR TITLE
Creating label for new node

### PR DESCRIPTION
**Pull request by Priyankakore21**

### CODE REVIEW

File: src/browser/icons/tick.svg [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
+<g>
+	<g>
+		<path d="M504.502,75.496c-9.997-9.998-26.205-9.998-36.204,0L161.594,382.203L43.702,264.311c-9.997-9.998-26.205-9.997-36.204,0
+			c-9.998,9.997-9.998,26.205,0,36.203l135.994,135.992c9.994,9.997,26.214,9.99,36.204,0L504.502,111.7
+			C514.5,101.703,514.499,85.494,504.502,75.496z"/>
+	</g>
+</g>
+<g>
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/icons/tick.svg]: why all these empty groups in here?

**Comment by rahul-tarafdar-synerzip** [comment]
it has conflicts

**Comment by Priyankakore21** [comment]
> it has conflicts

Resolved conflicts

**Comment by rahul-tarafdar-synerzip** [comment]
Thanks for recompose usage.
However, replace recompose with hooks

File: package.json [hunk 2]
```diff
@@ -144,6 +144,7 @@
     "react-suber": "1.0.4",
     "react-svg-inline": "^2.1.1",
     "react-timeago": "^4.1.9",
+    "recompose": "^0.30.0",
```
Comment (rahul-tarafdar-synerzip) [inline package.json]: remove from package.json

File: src/browser/modules/EditorInfo/Node.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import { TextInput } from 'browser-components/Form'
+
+function Node (props) {
+  const [textField, handleToggle] = useState(false)
+  const [nodeLabel, handleChange] = useState(undefined)
+  return (
+    <React.Fragment>
+      <NewNodeButton onClick={() => handleToggle(!textField)} />
+      {textField ? (
+        <Drawer>
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/Node.jsx]: why drawer when we have one already?

**Comment by rahul-tarafdar-synerzip** [comment]
i would like to see it running on your machine, first

**Comment by Priyankakore21** [comment]
> i would like to see it running on your machine, first

Ok

**Review by rahul-tarafdar-synerzip — APPROVED** [review]

**Pull request merged**